assembl/static2/js/app/pages/surveyAdmin.jsx

```diff
@@ -65,6 +82,7 @@ class SurveyAdmin extends React.Component {
 
       runSerial(mutationsPromises)
         .then(() => {
+          this.setState({ refetching: true });
```

Comment:
Don't you need to set again refatching to false?
`refetchThematics().then(() => this.setState({ refetching: false }));` ?

Reply:
you're right, it's probably better (even if there is no visible problem as it is)